testsuite/features/support/twopence_init.rb

```diff
@@ -84,14 +84,14 @@
   next if node.nil?
 
   hostname, _local, _remote, code = node.test_and_store_results_together('hostname', 'root', 500)
-  raise "Cannot get hostname for node. Response code: #{code}" if code.nonzero? || hostname.empty?
+  raise "Cannot connect to #{node} to get its hostname. Response code: #{code}" if code.nonzero? || hostname.empty?
```

Comment:
I think this is not really right, because we have two possibilities.

One is: you could connect.

The other is: you could connect, but the hostname was empty.

This should change to something like:

```
 raise "Cannot connect to #{node} to get its hostname. Response code: #{code}" if code.nonzero?
 raise "No hostname for #{node}. Response code: #{code}" if hostname.empty?
```
And the same for FQDN below.

Reply:
Agreed, I will modify it.

Reply:
done